larky/pom.xml

```diff
@@ -194,37 +206,20 @@
 
         <!-- test -->
         <dependency>
-              <groupId>org.junit.jupiter</groupId>
-              <artifactId>junit-jupiter-api</artifactId>
-              <version>5.8.1</version>
-              <scope>test</scope>
-          </dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.8.2</version>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>org.junit.vintage</groupId>
             <artifactId>junit-vintage-engine</artifactId>
-            <version>5.8.1</version>
+            <version>5.8.2</version>
             <scope>test</scope>
         </dependency>
     </dependencies>
 
     <build>
-        <resources>
-            <resource>
-                <directory>src/main/java</directory>
-            </resource>
-            <resource>
-                <directory>src/main/resources</directory>
-            </resource>
-            <resource>
-                <directory>src/test/java</directory>
-            </resource>
-            <resource>
-                <directory>src/test/resources</directory>
-                <excludes>
-                    <exclude>META-INF/services/*</exclude>
-                </excludes>
-            </resource>
-        </resources>
```

Comment:
Why though?

Reply:
This is not needed. Maven will automatically use our proper setup, so this isn't needed.